solutions/usaco-831.mdx

```diff
@@ -170,4 +171,69 @@ int main() {
 ```
 
 </CPPSection>
+
+<JavaSection>
+
+```java
+import java.io.*;
+import java.util.*;
+
+public class TeamTicTacToe {
+	private static char[][] gameboard = new char[3][3];
+	private static Set<Character> singleCow = new TreeSet<>();
+	private static Set<String> teams = new TreeSet<>();
+
+	private static void check(char first, char second, char third) {
+		// Stores unique cows in a row/column/diagonal
+        	Set<Character> contained = new TreeSet<>();
+        	contained.add(first);
+        	contained.add(second);
+        	contained.add(third);
```

Comment:
spaces -> tabs, `contained` actually has to be type TreeSet my bad
```suggestion
		TreeSet<Character> contained = new TreeSet<>();
		contained.add(first);
		contained.add(second);
		contained.add(third);
```

Reply:
Actually, I'm already using tabs, but it seems that there's some issue when I copy-paste my code from IDE to GitHub, I'll reformat it again.